<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>779</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>13</y>
     <width>981</width>
     <height>671</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <widget class="VideoLabel" name="labelVideo5">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>190</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="VideoLabel" name="labelVideo2">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>190</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="VideoLabel" name="labelVideo3">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>190</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="VideoLabel" name="labelVideo4">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>190</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="VideoLabel" name="labelVideo6">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>190</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="VideoLabel" name="labelVideo1">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>190</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="VideoLabel" name="labelVideo7">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>190</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="VideoLabel" name="labelVideo8">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>190</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="VideoLabel" name="labelVideo9">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>190</height>
       </size>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>690</y>
     <width>801</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>通道1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>通道2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>通道3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>通道4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>通道5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>通道6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>通道7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>通道8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>通道9</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Start">
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Stop">
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_2">
      <item>
       <property name="text">
        <string>9画面</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4画面</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1画面</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoLabel</class>
   <extends>QLabel</extends>
   <header location="global">video_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
